src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
   // your code here
+  this.name = name;
+  this.age = age;
+  return this;
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  return object[property]? true : false;
 };
 
 const isOver65 = person => {
   // your code here
+  return person.age > 65? true : false;
 };
 
 const getAges = people => {
   // your code here
+  return people.map(ageArray => ageArray.age);
 };
 
 const findByName = (name, people) => {
   // your code here
+  return people.find(person => person.name === name);
 };
 
 const findHondas = cars => {
   // your code here
+  return cars.filter(carModel => carModel.manufacturer === 'Honda');
```

Comment:
This is great. I would just suggest naming the variable `car` instead of `carModel` to be more precise as the variable actually contains more than just information about the model.